src/applications/personalization/profile360/actions/paymentInformation.js

```diff
@@ -28,7 +28,11 @@ export function savePaymentInformation(fields) {
   return async dispatch => {
     const apiRequestOptions = {
       headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify(fields),
+      body: JSON.stringify({
+        ...fields,
+        // eslint-disable-next-line no-undef
+        gaClientId: ga.getAll()[0].get('clientId'),
```

Comment:
I think I saw this in use somewhere else - maybe it would make sense to move it into a platform function?

Reply:
@ncksllvn , I can open a hardening ticket to address this tech-debt...

I'll pose my implementation question(s) there once created.